server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -233,6 +252,7 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
 
         kafkaCheckpointMessage = this.getKafkaCheckpointMessage(rawMessage);
         const checkpoint = this.generateCheckpoint(rawMessage, kafkaCheckpointMessage);
+        this.setDeliStateMetrics(checkpoint, lumberJackMetric);
```

Comment:
Could this be provided around an `if(lumberJackMetric) { }` statement. That way it won't create a `new Map` while disabled